package.json

```diff
@@ -66,6 +66,7 @@
     "circular-deps": "yarn madge --circular .",
     "release": "yarn ts-node ./scripts/release.ts release",
     "hs": "yarn build && node ./dist/bin/hs",
+    "unreleased-changes": "./scripts/unreleasedChanges.js",
```

Comment:
Is there any way that this could live in the `.github/` or `.github/workflows/` folders? It's not a blocker, but this isn't a script than can be run manually by us so it might be helpful to keep it separate.

Reply:
Not 100 percent certain, I'll give it a shot and see if it works out

Reply:
It took a lot of trial and error, but I moved the script into the `.github` directory and remove this line from the `package.json` and we are just running it using `node .github/scripts/unreleasedChanges.js`